src/tests/ftest/cart/rpc/multisend_one_node.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+'''
+  (C) Copyright 2018-2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+from cart_utils import CartTest
+
+
+class CartMultisendOneNodeTest(CartTest):
+    # pylint: disable=too-few-public-methods
+    """Run multisend test that launches 16 servers and splits bulk
```

Comment:
Can't add spelling errors.  So either you need to figure out a spelling that it won't complain about.  Or add pylint disable line for this warning.  My guess is multi-send would be ok.